frontend/src/components/cards/PlaceCard.js

```diff
@@ -17,6 +19,10 @@ const useStyles = makeStyles({
 export default function PlaceCard({ placeData }) {
   const classes = useStyles();
   const history = useHistory();
+  const latitude = placeData.latitude;
+  const longitude = placeData.longitude;
```

Comment:
Destructuring makes it short and sweet 
```suggestion
  const {latitude, longitude} = placeData;
```

Reply:
thx!